source/Public/New-GuestConfigurationPolicy.ps1

```diff
@@ -31,6 +31,14 @@
         The public HTTP or HTTPS URI of the Guest Configuration package (.zip) to run via the created policy.
         Example: https://github.com/azure/auditservice/release/AuditService.zip
 
+    .PARAMETER ContentManagedIdentity
+        This is the identity that is used to download the package from storage account container instead of using SaS url.
+        This value can be null.
```

Comment:
So - if you want to use managed identity instead of sas url, you need to use 3 parameters in unison - ContentUri (without sas token), ManagedIdentityResourceId and ContentPath.

ContentPath is the local package path to calculate hash and won't be directly added in the policy definition.